src/test/java/POMTests/FuelServicesTests.java

```diff
@@ -2,42 +2,88 @@
+        headerPage = new HeaderPage(driver);
+        searchResultsPage = new SearchResultsPage(driver);
+        laboratoryServicesPage = new LaboratoryServicesPage(driver);
+        fuelsAndBiocomponentsRequirementsPage = new FuelsAndBiocomponentsRequirementsPage(driver);
+        mainLocationPage = new MainLocationPage(driver);
+        footerPage = new FooterPage(driver);
+        cookiesViewPage = new CookiesViewPage(driver);
+        googleMapsPage = new GoogleMapsPage(driver);
+    }
 
 
     @Test
-    public void copyKoluszkiBaseAddress() throws InterruptedException {
-        MainPage mainPage = new MainPage(driver);
-        mainPage.goToFuelServicesPage().clickOnSelectedBase(774, 305, ".//a[@class = 'locations-map-list__url' and @href = 'https://www.pern.pl/obiekty/baza-paliw-nr-1-w-koluszkach/']");
-        BasesLocationDetailsPage basesLocationDetailsPage = new BasesLocationDetailsPage(driver);
-        String value = basesLocationDetailsPage.copyBaseAddress();
-        Assertions.assertEquals("ul. Naftowa 1, 95-040 Koluszki, powiat łódzki-wschodni, województwo łódzkie", value, "String value is not the same as needed");
-        Thread.sleep(3000);
+    public void goToKoluszkiBaseDetailsPage() {
+        //when
+        mainPage.goToFuelServicesPage().selectKoluszkiBaseLocation();
+        //then
+        Assertions.assertTrue(basesLocationDetailsPage.isKoluszkiDetailDisplayed());
     }
+
     @Test
-    public void goToFuelsPageAndClickOnKoluszkiBase(){
-        MainPage mainPage = new MainPage(driver);
-        mainPage.goToFuelServicesPage().clickOnSelectedBase(774, 305, ".//a[@class = 'locations-map-list__url' and @href = 'https://www.pern.pl/obiekty/baza-paliw-nr-1-w-koluszkach/']");
-        Assertions.assertEquals("https://www.pern.pl/obiekty/baza-paliw-nr-1-w-koluszkach/", driver.getCurrentUrl(), "URL of website is incorrect");
+    public void isAnyResultOfSearch() {
+        //given
+        headerPage.clickOnSearchBtn();
+        //when
+        headerPage.searchInSomeText(testDataReader.getSearchSafetyPhrase().getSearchSafetyValueInPERN());
+        //then
+        Assertions.assertTrue(searchResultsPage.isAnyResultNumber());
+
     }
+
     @Test
-    public void goToLaboratoryPageAndCheckDieselFlashPointValue(){
-        MainPage mainPage = new MainPage(driver);
-        mainPage.goToLaboratoryServicesPage();
-        LaboratoryServicesPage laboratoryServicesPage = new LaboratoryServicesPage(driver);
+    public void navigateToPlockLocation() {
+        mainPage.scrollToBottomMap().clickOnPlockLocation();
+        Assertions.assertEquals(testDataReader.getSection().getMainLocationSectionURL(), driver.getCurrentUrl());
+        mainLocationPage.clickOnNavigateBtn();
+        cookiesViewPage.closeCookiesBtn();
+        String inputValueFromGoogleMapsPage = googleMapsPage.getSearchInputValue();
+        Assertions.assertTrue(inputValueFromGoogleMapsPage.contains(testDataReader.getPernAddressInputValue().getAddress()));
```

Comment:
I prefer to user assertThat(), and probably you can import assert true directly without using Assertions.